orga/translations/fr.json

```
@@ -87,6 +87,21 @@
       },
       "loader": "Chargement des proportions par statut",
       "title": "Statuts"
+    },
+    "participants-by-day": {
+      "labels-a11y": {
+        "date": "Date",
+        "started": "Participants en cours de test",
+        "shared": "Participants ayant envoyés leurs résultats",
+        "shared-profile": "Participants ayant envoyés leurs profils"
```

J'ai regardé et c'est du gérondif passé (ayant + verbe au participe passé) - https://la-conjugaison.nouvelobs.com/regles/conjugaison/gerondif-passe-50.php - et les règles d'accord de l'auxiliaire avoir dit que ça doit être sans s : https://la-conjugaison.nouvelobs.com/regles/orthographe/l-accord-du-participe-passe-employe-avec-l-auxiliaire-avoir-179.php
```suggestion
        "shared": "Participants ayant envoyé leurs résultats",
        "shared-profile": "Participants ayant envoyé leurs profils"
```